runtime/lib/app-runtime.js

```diff
@@ -77,6 +77,7 @@ AppRuntime.prototype.init = function init () {
   this.componentsInvoke('init')
   /** set turen to not muted */
   this.component.turen.toggleMute(false)
+  this.component.turen.toggleWakeUpEngine(true)
```

Comment:
Should we move to `turen-component`'s init function?

Reply:
these two message should be sent after `flora` inited.

Reply:
Ok, maybe we could do that when the component stage is done.